### ISSUE TITLE
Reduced reactive oxygen species production in neutrophils

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11126]
**HPO term**
[Reduced reactive oxygen species production in neutrophils](https://hpo.jax.org/browse/term/HP:0025632)

Is this different from Decreased neutrophil oxidative burst or should it be merged?

**Comment by adamklocperk** [comment]
@pnrobinson no, these are the same. We mildly prefer our label, but both are adequate.

**Comment by pnrobinson** [comment]
OK, I merged this term into Decreased neutrophil oxidative burst, and added Reduced reactive oxygen species production in neutrophils as a synonym. Thanks!
